great_expectations/expectations/set_based_column_map_expectation.py

```diff
@@ -58,13 +76,33 @@ def _spark(cls, column, **kwargs):
 
 
 class SetBasedColumnMapExpectation(ColumnMapExpectation, ABC):
+    """Base class for SetBasedColumnMapExpectations.
+
+    SetBasedColumnMapExpectations facilitate set-based comparisons as the core logic for a Map Expectation.
+
+    Attributes:
```

Comment:
For some reason the rendering doesn't like `Attributes:`
```suggestion
    Args:
```